src/manifolds/ProbabilitySimplex.jl

```diff
@@ -281,6 +281,35 @@ mean(::ProbabilitySimplex, ::Any...)
 
 default_estimation_method(::ProbabilitySimplex, ::typeof(mean)) = GeodesicInterpolation()
 
```

Comment:
We usually document our functions on the non-mutating case, i.e. here `rand()`, see https://github.com/JuliaManifolds/Manifolds.jl/blob/369eef7418089baaf59a6215ed2fd94b23174d8a/src/manifolds/Stiefel.jl#L310-L322 – could you add that here as well? If possible, a formula / name / reference would be great as well